I'm getting the same error of @travis-ci

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:packageDebug'.
> Duplicate files copied in APK META-INF/NOTICE
    File 1: /Users/passos/Development/Projects/FeedHenry-Template/welcome-android-gradle/app/build/intermediates/exploded-aar/com.feedhenry/fh-android-sdk/3.1.0/jars/classes.jar
    File 2: /Users/passos/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpclient-android/4.3.5.1/eecbb0b998e77629862a13d957d552b3be58fc4e/httpclient-android-4.3.5.1.jar


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -21,6 +21,9 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    packagingOptions {
+        exclude 'META-INF/NOTICE'
+    }
 }
 
 dependencies {

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -23,6 +23,7 @@ android {
     }
     packagingOptions {
         exclude 'META-INF/NOTICE'
+        exclude 'META-INF/LICENSE'
     }
 }
 

